packages/react-app/src/components/bridge/ToToken.jsx

```diff
@@ -154,6 +188,19 @@ export const ToToken = () => {
                 {utils.formatUnits(amount, token.decimals)}
               </Text>
             )}
+            {bridgeDirection !== ETH_XDAI_BRIDGE && // Temporary block and will be removed once the audit is finished.
+              WRAPPED_CURRENCY_SYMBOLS.includes(fromToken.symbol) &&
+              NATIVE_CURRENCY_CHAIN_IDS.includes(chainId) && (
+                <Flex>
+                  <Text>Receive Native Currency</Text>
```

Comment:
Consider replace `Native Currency` with the actual currency symbol. Such as `Receive KETH` or `Receive BNB` as per the bridge direction.